app/rental_routes.py

```diff
@@ -0,0 +1,103 @@
+import re
+from app import customer_routes, db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from datetime import date, datetime, timedelta
+from flask import Blueprint, jsonify, request
+
+
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+
+@rental_bp.route("/check-out", methods=["POST"])
+def rental_check_out():
+    request_body = request.get_json()
+
+    if "customer_id" not in request_body:
+        return jsonify(None), 400
+    if "video_id" not in request_body:
+        return jsonify("Could not perform checkout"), 400
+    
+    customer = Customer.query.get(request_body["customer_id"])
+    video = Video.query.get(request_body["video_id"])
+
+    if hasattr(video, 'video_rentals') is False:
+        return jsonify(None), 404
+    if hasattr(customer, 'customer_rentals') is False:
+        return jsonify(None), 404
+    count_of_rentals = len(video.video_rentals)
+
+    video_avialable_inventory = video.total_inventory - count_of_rentals
```

Comment:
another suggestion would be to add `video_checkout_count = db.Column(db.Integer, default=0` to you Customer model then add to the count and subtract from the inventory count. for videos
```suggestion
     customer.videos_checkout_count += 1
    video.total_inventory -= 1
```